packages/contracts/contracts/Unirep.sol

```diff
@@ -100,18 +103,14 @@ contract Unirep is IUnirep, VerifySignature {
             if (initialData[x] >= SNARK_SCALAR_FIELD) revert InvalidField();
             if (
                 x >= sumFieldCount &&
-                (x - sumFieldCount) % 2 == (sumFieldCount + 1) % 2 &&
-                initialData[x] != 0
-            ) {
-                revert InvalidTimestamp();
-            }
+                initialData[x] >= 2 ** (254 - replNonceBits)
+            ) revert OutOfRange();
             emit Attestation(
-                epoch,
+                type(uint48).max,
```

Comment:
When the user signs up with data we attest to the identity commitment. The attestation doesn't go in the epoch tree though (because the data is already in the state tree leaf). This makes it difficult to determine the index of other attestations in the epoch tree.

By putting it at the final epoch we can still process it by querying for `{ epochKey: identityCommitment }` but it doesn't cause problems in the current epoch.